mgmtsystem_action/models/mgmtsystem_action_report.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models
+from openerp import tools
+
+
+class MgmtsystemtActionReport(models.Model):
+    """Management System Action Report."""
```

Comment:
Yep!

Maxime Chambreuil
Le 26 mai 2016 12:18, "Daniel Reis" <notifications@github.com> a écrit :

> In mgmtsystem_action/models/mgmtsystem_action_report.py
> <https://github.com/OCA/management-system/pull/115#discussion_r64784552>:
>
> > @@ -0,0 +1,70 @@
> > +# -*- coding: utf-8 -*-
> > +
> > +from openerp import fields, models
> > +from openerp import tools
> > +
> > +
> > +class MgmtsystemtActionReport(models.Model):
> > +    """Management System Action Report."""
>
> IMO this should go inside the reports directory.
> @max3903 <https://github.com/max3903> is it so?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/management-system/pull/115/files/36cb8e6f85fcccd3f1824d530fbbe8021f2fb633#r64784552>
>